hyrisecockpit/database_manager/worker.py

```diff
@@ -133,6 +128,14 @@ def execute_queries(
                 ) as e:
                     failed_queries.append((time_ns(), worker_id, str(task), str(e)))
 
-                last_batched = log_results(
-                    log, last_batched, succesful_queries, failed_queries
-                )
+                if last_batched < time_ns() - 1_000_000_000 and (
+                    succesful_queries != [] or failed_queries != []
```

Comment:
https://docs.python.org/3.9/library/stdtypes.html#truth-value-testing
```suggestion
                    succesful_queries or failed_queries
```